compiler/noirc_frontend/src/hir/def_map/mod.rs

```diff
@@ -295,7 +295,7 @@ pub struct Contract {
 
 /// Given a FileId, fetch the File, from the FileManager and parse it's content
 pub fn parse_file(fm: &FileManager, file_id: FileId) -> (ParsedModule, Vec<ParserError>) {
-    let file_source = fm.fetch_file(file_id);
+    let file_source = fm.fetch_file(file_id).expect("File does not exist");
```

Comment:
This seems like the only dodgy `expect`, however we can just make this part of the contract of this function.